src/matrix/matrixtdist.jl

```diff
@@ -0,0 +1,194 @@
+"""
+```julia
+MatrixTDist(ν, M, Σ, Ω)
+
+ν:  positive Real
+M:  n x p matrix
+Σ:  n x n positive definite matrix
+Ω:  p x p positive definite matrix
+```
+The [Matrix *t*-Distribution](https://en.wikipedia.org/wiki/Matrix_t-distribution)
+generalizes the Multivariate *t*-Distribution from vectors to matrices. An
+``n\\times p`` matrix ``\\mathbf{X}`` with the matrix *t*-distribution has density
+
+```math
+f(\\mathbf{X} ; \\nu,\\mathbf{M},\\boldsymbol{\\Sigma}, \\boldsymbol{\\Omega}) =
+c_0 \\left|\\mathbf{I}_n + \\boldsymbol{\\Sigma}^{-1}(\\mathbf{X} - \\mathbf{M})\\boldsymbol{\\Omega}^{-1}(\\mathbf{X}-\\mathbf{M})'\\right|^{-\\frac{\\nu+n+p-1}{2}},
+```
```

Comment:
This will not render in the REPL... can we just rewrite it in plain text, using unicode as needed?

Reply:
Right. I'm emulating, for instance, [this](https://github.com/JuliaStats/Distributions.jl/blob/6561a7e00132c6698a961d454219b915bfb789f6/src/univariate/continuous/fdist.jl#L1) and [this](https://github.com/JuliaStats/Distributions.jl/blob/6561a7e00132c6698a961d454219b915bfb789f6/src/multivariate/mvnormal.jl#L25).

Reply:
hmm, i see the precedent, but i cannot help but think it is unfortunate, as I think docstrings should be readable from the REPL whenever possible -- that seems the most user-friendly

Reply:
I agree completely. But since the docstring is doing double duty, I don't know how to get around it. In hindsight, [this](https://juliastats.github.io/Distributions.jl/latest/matrix/#Distributions.MatrixNormal) is kind of horrific, but works in the REPL. I will change it very soon.